Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -78,75 +108,514 @@ public static void ShowWindow()
             EditorGUILayout.LabelField(dialogTitle, EditorStyles.boldLabel);
+            CreateSpace(15);
             EditorGUILayout.LabelField(dialogContent);
 
+            CreateSpace(10);
+            if (GUILayout.Button(ApplyButtonContent))
+            {
+                ApplyConfigurations();
+            }
+
+            RenderSetupLaterSection(true, () => {
+                currentStage = ConfigurationStage.ImportTMP;
+                Repaint();
+            });
+        }
+
+        private void RenderImportTMP()
+        {
+            if (TMPEssentialsImported())
+            {
+                currentStage = ConfigurationStage.ShowExamples;
+                Repaint();
+            }
+            EditorGUILayout.LabelField("Importing TMP Essentials", EditorStyles.boldLabel);
+            
+            EditorGUILayout.LabelField("MRTK contains components that depend on TextMeshPro. It is recommended that you import TMP by clicking the Import TMP Essentials button below.", EditorStyles.wordWrappedLabel);
+            CreateSpace(15);
+            var m_ResourceImporter = new TMP_PackageResourceImporter();
+            m_ResourceImporter.OnGUI();
+            CreateSpace(15);
+            RenderSetupLaterSection(true, () => {
+                currentStage = ConfigurationStage.ShowExamples;
+                Repaint();
+            });
+        }
+
+        private void RenderShowUPMExamples()
+        {
+            if (!MRTKExamplesPackageImportedViaUPM())
+            {
+                currentStage = ConfigurationStage.Done;
+                Repaint();
+            }
+            EditorGUILayout.LabelField("Locating MRTK Examples", EditorStyles.boldLabel);
+            
+            EditorGUILayout.LabelField("The MRTK Examples package includes samples to help you familiarize yourself with many core features. "
+                + "\nSince you imported MRTK via MRFT/UPM the examples no longer show up in the Assets folder automatically. They are now located at Window (menu bar) -> Package Manager "
+                + "-> Select In Project in the \"Packages:\" dropdown -> Mixed Reality Toolkit Examples", EditorStyles.wordWrappedLabel);
+            CreateSpace(15);
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                if (GUILayout.Button("Show me the examples"))
+                {
+#if UNITY_2019_3_OR_NEWER
+                    UnityEditor.PackageManager.UI.Window.Open("Mixed Reality Toolkit Examples");
+#endif
+                }
+                if (GUILayout.Button("Got it, next"))
+                {
+                    currentStage = ConfigurationStage.Done;
+                    Repaint();
+                }
+            }
+            CreateSpace(15);
+            RenderSetupLaterSection();
+        }
+
+        private void RenderConfigurationCompleted()
+        {
+            EditorGUILayout.LabelField("MRTK Setup Completed!", MixedRealityStylesUtility.BoldLargeTitleStyle);
```

Comment:
```suggestion
                GUILayout.Label("MRTK Setup Completed!", MixedRealityStylesUtility.BoldLargeTitleStyle);
```
EditorGUILayout causes the text to be cut off oddly, use the following to fix the display issues